_includes/tutorials/splitting/confluent/markup/dev/ccloud-run-producer.adoc

```diff
@@ -0,0 +1,11 @@
+In a new terminal, run:
```

Comment:
Or my SR comment might apply here.

Reply:
Added in commit https://github.com/confluentinc/kafka-tutorials/pull/832/commits/2f540214b5f9580bd73f7cb65725f71d200b1664